gradle-plugin/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'java'
+    id 'java-gradle-plugin'
+    id 'com.diffplug.spotless' version("6.10.0")
+}
+
+version = '0.1'
+group = 'com.contrastsecurity.gradle.plugin'
+
+repositories {
+    mavenCentral()
+}
+
+gradlePlugin {
+    plugins {
+        java {
+            id = 'com.contrastsecurity.java'
```

Comment:
How are we choosing the plugin name?

Reply:
I forgot to bring this up, no idea 🙃 nobody picked a name yet, this is completely arbitrary. Easily changeable at least, I might make permanently choosing that a requirement for before we actually release

Reply:
Yeah, it's not something we need to decide until we publish. Calling the plugin "java" seems a bit vague, but I can't think of something better at the moment. 🤷